train/training.py

```diff
@@ -93,6 +94,61 @@ def convert_labels_to_csv(data, tagging_output_file_path):
         print("An error occurred attempting to write to file at {0}:\n\n{1}".format(tagging_output_file_path,e))
         raise
 
+def upload_data_post_training(training_id):
+    tagging_output_file_path =  os.path.expanduser(config.get('tagged_output'))
+    function_url = config.get("url") + "/api/classification"
+    query = {
+        "userName": config.get('tagging_user')
+    }
+
+    # First, we need to get a mapping of class names to class ids
+    classes = ""
+    with open(tagging_output_file_path + "/tagged.csv") as f:
+        content = f.readlines()
+        for line in content:
+            class_name = line.strip().split(',')[1]
+            if class_name not in classes:
+                classes = classes + class_name + ","
+    
+    query["className"] = classes[:-1]
+    response = requests.get(function_url, params=query)
+    classification_name_to_class_id = response.json()
+    
+    # Now that we have a mapping, we create prediction labels in db
+    query = {
+        "userName": config.get('tagging_user'),
+        "trainingId": training_id
+    }
+    function_url = config.get("url") + "/api/labels"
+    payload_json = process_post_training_csv(tagging_output_file_path + "/tagged.csv", training_id, classification_name_to_class_id)
+    requests.post(function_url, params=query, json=payload_json)
+
+def process_post_training_csv(csv_path, training_id, classification_name_to_class_id):
+    payload_json = []
+    with open(csv_path) as f:
+        content = f.readlines()
+        count = 0
+        for line in content:
+            if count > 0:
```

Comment:
@samiyaakhtar I assume we are doing this to skip the head row. Also we are manually splitting the lines by comma. An approach with less code could be:

```
import csv
with open(csv_path) as f:
         reader = csv.reader(f)
         next(reader, None) #Skip header
         for row in reader:
               ....
               # Use row[n] to access columns 
```

Reply:
Done